src/cli/project/channel/remove.rs

```diff
@@ -0,0 +1,87 @@
+use crate::environment::update_prefix;
+use crate::lock_file::{load_lock_file, update_lock_file};
+use crate::prefix::Prefix;
+use crate::Project;
+use clap::Parser;
+use itertools::Itertools;
+use miette::IntoDiagnostic;
+use rattler_conda_types::{Channel, ChannelConfig, Platform};
+
+/// Remove a channel(s) to the project file and updates the lockfile.
+#[derive(Parser, Debug, Default)]
+pub struct Args {
+    /// The channel name(s) or URL
+    #[clap(required = true, num_args=1..)]
+    pub channel: Vec<String>,
+
+    /// Don't update the environment, only remove the channel(s) to the lock-file.
+    #[clap(long)]
+    pub no_install: bool,
+}
+
+pub async fn execute(mut project: Project, args: Args) -> miette::Result<()> {
+    // Determine which channels are missing
```

Comment:
Not sure what you want to say with "missing", it seems that these are the channels you want to remove.

Reply:
Changed to `// Determine which channels to remove`